dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```

Comment:
not required, but should have the bndtools nature on the .project file (see example FAT for template)